debug/captions.html

```diff
@@ -319,13 +318,9 @@ <h3>footer</h3>
             }
 
             for (j = 0, i = offset; j < remuxedSegments.length; j++) {
-              var boxData;
-
-              if (!partial) {
-                boxData = remuxedSegments[j].data;
-              } else {
-                boxData = remuxedSegments[j].data.boxes;
-              }
+              var boxData = partial ?
```

Comment:
This can just be: `var boxData = partial ? remuxedSegments[j].data.boxes : remuxedSegments[j].data;`